docs/hive-integration/agent-labels.md

```diff
@@ -10,9 +10,41 @@ An annotation on the Agent CR indicates a version for the labels, so clients can
 Labels marked as boolean will have either the string "true" or "false".
 
 ### v0.1
-* feature.agent-install.openshift.io/storage-hasnonrotationaldisk (boolean): Indicates if the Agent has at least one SSD
-* feature.agent-install.openshift.io/cpu-architecture (string): The CPU architecture (e.g., x86_64, arm64)
-* feature.agent-install.openshift.io/cpu-virtenabled (boolean): Indicates if the CPU has the virtualization flag (VMX or SVM)
-* feature.agent-install.openshift.io/host-manufacturer (string): The host's manufacturer
-* feature.agent-install.openshift.io/host-productname (string): The host's product name
-* feature.agent-install.openshift.io/host-isvirtual (boolean): Indicates if the host is a virtual machine
\ No newline at end of file
+* inventory.agent-install.openshift.io/storage-hasnonrotationaldisk (boolean): Indicates if the Agent has at least one SSD
+* inventory.agent-install.openshift.io/cpu-architecture (string): The CPU architecture (e.g., x86_64, arm64)
+* inventory.agent-install.openshift.io/cpu-virtenabled (boolean): Indicates if the CPU has the virtualization flag (VMX or SVM)
+* inventory.agent-install.openshift.io/host-manufacturer (string): The host's manufacturer
+* inventory.agent-install.openshift.io/host-productname (string): The host's product name
+* inventory.agent-install.openshift.io/host-isvirtual (boolean): Indicates if the host is a virtual machine
+
+# Agent Classification labels
+
+The AgentClassification CRD defines the API for users to classify Agents by providing a query that is run on the Agent's inventory along with a corresponding label.  
+The query format is defined by the [gojq](https://github.com/itchyny/gojq) library, which supports jq queries in Go.
+Any Agent in the same namespace as the AgentClassification whose inventory causes the specified query to evaluate to `true` will have the specified label applied (prefixed by "agentclassification.agent-install.openshift.io/").
+
+Some examples include:
+
+```
+spec:
```

Comment:
Would be nice to have an entire CR with metadata, kind and everything so users get a better context for where this goes

Reply:
Good point, added to docs/hive-integration/crds/agentClassification.yaml